apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html

```diff
@@ -0,0 +1,953 @@
+                        <mat-grid-list cols="2" rowHeight="{{ rowLength }}px" gutterSize="0px" class="sidebar-box">
+                            <mat-grid-tile colspan="2">
+                                <div class="side-bar-items" (click)="selectSubscription(subscription)">
+                                    <p class="mat-body-2 side-bar-head">
+                                        {{ subscription?.planDetails?.name
+                                        }}<img src="{{ imgPath }}dot.png" alt="dot" /><span>{{
+                                            subscription?.subscriptionId
+                                        }}</span>
+                                    </p>
+                                    <div class="side-bar-list">
+                                        <img src="{{ imgPath }}transaction.png" alt="transaction" />
+                                        <p class="side-bar-list-title">{{ commonLocaleData?.app_transactions }}</p>
+                                        <span>:</span>
+                                        <p class="mat-body-2">
+                                            {{ subscription?.remainingTransactions }}
+                                            <span class="warning mat-body-2">{{ localeData?.subscription?.left }}</span>
+                                        </p>
+                                    </div>
+                                    <div class="side-bar-list">
+                                        <img src="{{ imgPath }}companies.png" alt="companies" />
+                                        <p class="mat-subheading-4 side-bar-list-title">
+                                            {{ localeData?.subscription?.companies }}
+                                        </p>
+                                        <span>:</span>
+                                        <p>{{ subscription?.totalCompanies | number }}</p>
+                                    </div>
+                                    <div class="side-bar-list plan-timeline">
+                                        <img src="{{ imgPath }}plan_timeline.png" alt="plan_timeline" />
+                                        <p class="mat-subheading-4 side-bar-list-title">
+                                            {{ localeData?.subscription?.plan_timeline }}
+                                        </p>
+                                        <span>:</span>
+                                        <p class="date">{{ subscription?.startedAt }} - {{ subscription?.expiry }}</p>
+                                    </div>
+                                </div>
+                            </mat-grid-tile>
+                        </mat-grid-list>
+                    </ng-container>
+                </div>
+                <div class="sidebar-clone" *ngIf="showLoader">
+                    <ng-container *ngFor="let subscriptionStatic of subscriptionsDummy">
```

Comment:
change subscriptionsDummy to array of string and put - in place of values since values won't be displaying

Reply:
DONE